web/Views/Shared/_Layout.cshtml

```diff
@@ -34,18 +34,20 @@
                 <div class="navbar-nav">
                     @if (User.Identity?.IsAuthenticated == true)
                     {
-                        @if (User.IsInRole("Landlord"))
-                        {
-                            <div class="d-flex align-items-center mx-3 me-sm-0 py-2 py-sm-0">
-                                <a asp-controller="Landlord" asp-action="Profile" asp-route-id="@(((BricksAndHeartsUser)User.Identity).LandlordId!.Value)" class="profile-photo-sm">
-                                    <img src="@(((BricksAndHeartsUser)User.Identity).GoogleProfileImageUrl ?? Url.Content("~/images/profileDefault.png"))" alt="Profile picture" referrerpolicy="no-referrer"/>
-                                </a>
-                            </div>
-                        }
-                        @using (Html.BeginForm("Logout", "Login", FormMethod.Post))
-                        {
-                            <button type="submit" class="btn btn-thin btn-outline-primary mx-3">Log out</button>
-                        }
+                        <div class="d-flex">
+                            @if (User.IsInRole("Landlord"))

```

Comment:
indentation is off here I thinkk

Reply:
I thought if we have a div in an if statement we were supposed to indent it? What would you change it to be?

Reply:
and similarly for if statements in div's

Reply:
ahhh ok thanks for clearing that up :)